tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -0,0 +1,242 @@
+import {generateTestProject} from "./utils/utils";
+import {execSync} from "child_process";
+import waitForExpect from "wait-for-expect";
+import fs from "fs";
+import {isPortReachable} from "./utils/utils";
+
+
+const taqueriaProjectPath = 'e2e/auto-test-flextesa-plugin';
+let dockerName: string;
+
+describe("E2E Testing for taqueria flextesa plugin",  () => {
+
+    beforeAll(async () => {
+        await generateTestProject(taqueriaProjectPath, ["flextesa"]);
+    })
+
+    test('Verify that taqueria flextesa plugin can start and stop a default sandbox without specifying name', async () => {
+        try {
+
+            // Setting up docker container name
+            dockerName = "local"
+
+            // 1. Run sandbox start command
+            const stdoutStart = execSync(`taq start sandbox`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 2. Verify that sandbox has been started and taqueria returns proper message into console
+            expect(stdoutStart).toEqual("Started local.");
+
+            // 3. Verify that docker container has been started
+            const [,dockerContainerLocal] = execSync(`docker ps --filter "name=${dockerName}"`).toString().trim().split(/\r?\n/);
+            expect(dockerContainerLocal).toBeDefined();
```

Comment:
Since you're splitting out the docker content from the labels I would say looking for something more concrete than it just being defined would be a better test. With the output being one giant string I would say that looking for the command that started the container would be a good idea. If you can verify that the command the container is running is something like `"node startFlextesa.js --sandbox ${dockerName}"` then that is a much stronger test

Reply:
@alexzbusko Hey Alex, let's discuss it because it is a bit unclear what you mean by "the command the container is running". Does `docker ps` serve this purpose?

Reply:
Fixed. added a more specific expect statement.